core/controllers/suggestion.py

```diff
@@ -139,6 +161,68 @@ def put(self, target_id, suggestion_id):
         self.render_json(self.values)
 
 
+class ReviewableSuggestionsHandler(base.BaseHandler):
+    """Provides all suggestion which can be reviewed by the user for a given
+    suggestion type.
+    """
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_access_admin_page
+    def get(self, target_type, suggestion_type):
+        """Handles GET requests."""
+        try:
+            _check_valid_suggestion_type_and_target_type(
+                target_type, suggestion_type)
+
+            suggestions = suggestion_services.get_reviewable_suggestions(
+                self.user_id, suggestion_type)
+
+            if target_type == suggestion_models.TARGET_TYPE_EXPLORATION:
+                target_id_to_details = (
+                    opportunity_services.get_opportunities_by_ids(
```

Comment:
For get_multi()-type calls, typically we return a list with the same number of elements, using None if the element cannot be found. Is that a feasible convention to adopt here?

Also, should we be deduplicating the input list? (Can there be multiple suggestions for one target ID?)

Reply:
> For get_multi()-type calls, typically we return a list with the same number of elements

I've followed the pattern used in exp_service (Which I think to have the most generic structure and I also find the dict with id as key quite useful.)

> Can there be multiple suggestions for one target ID?

Yep, there can be multiple translation suggestions for one exploration.

Reply:
I've fixed the issue now!